collect_app/src/test/java/org/odk/collect/android/formentry/QuitFormDialogTest.kt

```diff
@@ -80,9 +81,8 @@ class QuitFormDialogTest {
     }
 
     @Test
-    fun whenSaveAsDraftIsEnabled_andLastSavedTimeIsNull_showsSaveExplanation() {
+    fun whenSaveAsDraftIsEnabled_andFormCanBeFullyDiscarded_showsSaveExplanation() {
```

Comment:
I'd tend to still set up the correct state using `whenever` here so it's easier to see the state being used in the test (without having to scroll to the original set up).